fontforge/python.c

```diff
@@ -12290,6 +12305,8 @@ return( -1 );
 	    BaseFree(base);
 return( -1 );
 	}
+	bs = chunkalloc(sizeof(struct basescript));
```

Comment:
wow, I guess this never worked before?

Reply:
Correct. If I'm remembering right there are paths through the function that worked but anything that reached this line would have failed.

Reply:
Hmm, actually maybe only error-producing paths worked. 

Anyway a number of the bugs I've fixed had implementations that could never have worked. #3355 was another example in the same file.